feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -67,7 +67,6 @@
     <button
       #searchResultItem
       (click)="createCustomer()"
-      (keydown.enter)="createCustomer()"
```

Comment:
removed, When opening the Create New Customer dialog popup in the customer search box by pressing Enter on the keyboard, we expected that only the dialog will be opened without submitting the form.